### DISCUSSION TITLE
OneAPI的版本需要升级支持新版工具调用

### DISCUSSION THREAD

**Discussion by YersiniaHerb** [BurnHair/docs#208] [技术支持]
lobechat加工具节点连大模型代理报错
应该是不支持新版的调用方式
![image](https://github.com/user-attachments/assets/775026c6-ff7d-4ccd-a70a-f1de6f6bf14c)

**Comment by BennyThink** [comment]
☹️ 可以先试试 https://api.burn.hair/ 这个我可能得过一段时间才有时间修复

**Comment by BennyThink** [comment]
应该修好了，再试试看呢
